website/js/repl.js

```diff
@@ -167,26 +191,11 @@ function compile() {
       // turn off compiling
 
       var result = e.data;
-      if (result.type === 'success') {
+      if (result.type === 'success' || result.type === 'warning') {
```

Comment:
There is really no functional difference between 'success' and 'warning', right? Let's just stick to 'success' then for all non-error results.

Reply:
Okay ! :+1: